### DISCUSSION TITLE
Error while installation

### DISCUSSION THREAD

**Discussion by assang** [Siteimprove/alfa#608] [Questions] [answered]
Hi Devs,

I am unable to install using npm install @siteimprove/alfa-act or any other package

**Gets Error** I have added .npmrc file but still gives this error. Do I need any credentials or configuration?
npm ERR! code UNABLE_TO_GET_ISSUER_CERT_LOCALLY
npm ERR! errno UNABLE_TO_GET_ISSUER_CERT_LOCALLY
npm ERR! request to https://npm.pkg.github.com/@siteimprove%2falfa-act failed, reason: unable to get local issuer certificate

**Comment by kasperisager** [comment, marked as answer]
Hi there and thanks for reaching out! It sounds like you haven't yet set up authentication for GitHub Packages, which is currently required to consume both public and private packages: https://docs.github.com/en/free-pro-team@latest/packages/guides/configuring-npm-for-use-with-github-packages#authenticating-to-github-packages

**Comment by kasperisager** [comment]
Did you manage to set up authentication, @assang?

**Comment by assang** [comment]
No it is giving other error. (I have added taken as @siteimprove:registry=https://npm.pkg.github.com/:_authToken=************************** )

Now Error is:
npm ERR! code E401
npm ERR! Unable to authenticate, need: Basic realm="GitHub Package Registry"

npm ERR! A complete log of this run can be found in:

**Comment by kasperisager** [comment]
Did you add the token by doing `npm login --scope=@siteimprove --registry=https://npm.pkg.github.com`? If so, I imagine that the token provided might simply miss the necessary permissions; at a minimum, it needs the `read:packages` scope:

![image](https://user-images.githubusercontent.com/1174718/102197688-c46cb180-3ec1-11eb-87b6-70c17e3fe9f5.png)

**Comment by assang** [comment]
I tried the same steps again and now getting error. Does it mean I need some permission from npm repo or alfa repo?

Not Found - PUT https://npm.pkg.github.com/:_authToken=51ec209133ec45e874d1def3e674f277edae3cbe/-/user/org.couchdb.user:assang       
npm ERR! 404
npm ERR! 404  'org.couchdb.user:assang' is not in the npm registry.
npm ERR! 404 Your package name is not valid, because
npm ERR! 404  1. name can only contain URL-friendly characters

**Comment by kasperisager** [comment]
The issue lies with authentication on your end as there's nothing needed from our end for you to consume public packages from the GitHub npm registry. If you're in need of more help with authentication beyond what the GitHub docs provide, I'd recommend reaching out to the Support Community: https://github.community/c/github-help/48
